scripts/lib/CIME/case_setup.py

```diff
@@ -37,7 +37,11 @@ def _build_usernl_files(case, model, comp):
     multi_driver = case.get_value("MULTI_DRIVER")
     if multi_driver:
         ninst_max = case.get_value("NINST_MAX")
-        if model not in ("DRV","CPL"):
+# KDR added ESP so that I can build with 1,
+#     which will make st_archive work with .dart.$string files,
+#     which have no instance number.
+#     Should ESP be considered a data component?
+        if model not in ("DRV","CPL","ESP"):
```

Comment:
This comment should be trimmed or eliminated
Do not need initials (use git blame for that) and remove question.
I support just removing the comment since I do not think we have figured out if ESP models can run in multi_driver mode and what happens if they do so we can fix this later if needed.

Reply:
I'll remove the comment, but I need to leave the fix in
because it's needed to use MULTI_DRIVER and have
st_archive work on the DART output files.  
Or I need some other fix, because
MULTI_DRIVER will be the standard way to run DART cases 
because it's so much more efficient for typical ensemble sizes.

Reply:
Yes, all I am asking is that the comment be removed.